tests/README.md

```diff
@@ -0,0 +1,119 @@
+# System Testing
+
+The tests in this directory are meant to be run on a live Kubernetes environment to verify a real system. These
+are similar to the existing [conformance tests](../conformance/README.md), but will verify things such as:
+
+- NGF-specific functionality
+- performance
```

Comment:
instead of performance and scale, would it make sense to have Non-Functional requirements testing, because it includes performance, scale, and more things? for example, reliability, availability